### PR TITLE
Backport the 5.2 contribution guide to 5.1

### PR DESCRIPTION
**Pull request by GrahamCampbell**

### CODE REVIEW

File: contributions.md [hunk 1]
```diff
@@ -31,12 +32,14 @@ The Laravel source code is managed on Github, and there are repositories for eac
...
+You may propose new features or improvements of existing Laravel behavior in the Laravel Internals [issue board](https://github.com/laravel/internals/issues). If you propose a new feature, please be willing to implement at least some of the code that would be needed to complete the feature.
+
+Informal discussion regarding bugs, new features, and implementation of existing features takes place in the `#internals` channel of the [LaraChat](http://larachat.co) Slack team. Taylor Otwell, the maintainer of Laravel, is typically present in the channel on weekdays from 8am-5pm (UTC-06:00 or America/Chicago), and sporadically present in the channel at other times.
 
 <a name="which-branch"></a>
 ## Which Branch?
 
-**All** bug fixes should be sent to the oldest stable branch where the fix is applicable. Bug fixes should **never** be sent to the `master` branch unless they fix features that exist only in the upcoming release.
+**All** bug fixes should be sent to the latest stable branch or to the current LTS branch (5.1). Bug fixes should **never** be sent to the `master` branch unless they fix features that exist only in the upcoming release.
```
Comment (vlakoff) [inline contributions.md]: I prefer my previous wording. Less ambiguous. :relaxed:

Comment (GrahamCampbell) [inline contributions.md]: oldest stable is incorrect though

Comment (GrahamCampbell) [inline contributions.md]: that would be 3.x

Comment (vlakoff) [inline contributions.md]: Yeah I had tried things like "oldest maintained branch" but found them not as fluent.
"oldest stable" is not that bad, if you consider "stable" excludes "deprecated".

Though, I'd still vote for "oldest maintained branch", to be the clearest possible.

**Pull request merged**